<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New User</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/icons/calendar icon.svg</normaloff>:/resource/icons/calendar icon.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="firstname_label">
       <property name="text">
        <string>First Name *</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="firstname_input"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lastname_label">
       <property name="text">
        <string>Last Name *</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lastname_input"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="username_label">
       <property name="text">
        <string>Username *</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="username_input"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>Password *</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="password_input"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="email_label">
       <property name="text">
        <string>E-mail *</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="emal_input"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="emailpassword_label">
       <property name="text">
        <string>E-mail Password *</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="emailpassword_input"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="register_button">
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>emal_input</tabstop>
  <tabstop>emailpassword_input</tabstop>
  <tabstop>firstname_input</tabstop>
  <tabstop>lastname_input</tabstop>
  <tabstop>username_input</tabstop>
  <tabstop>password_input</tabstop>
  <tabstop>register_button</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
